style.css

```diff
@@ -240,3 +241,119 @@ hr {
   width: 303px;
   padding: 10px 0 10px 10px;
 }
+
+/* Contact form */
+
+.contact {
+  width: 370px;
+  height: 779px;
+  background: #ffff;
+}
+
+.contact-form {
+  background: #6070ff;
+  border: none;
+  width: 370px;
+  height: 789px;
+  margin-left: -1px;
+  border-top-left-radius: 80px;
+  padding: 0;
+}
+
+.background-img {
+  background-image: url("images/contact-form-mobile.svg");
+  height: 709px;
+  background-repeat: no-repeat;
+  background-position: right;
+  margin-top: 30px;
+}
+
+.contact-heading {
+  color: #fff;
+  text-align: center;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  position: absolute;
+  margin-top: 80px;
+  width: 345px;
+}
+
+.contact-details {
+  color: #ebebff;
+  text-align: center;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  margin-top: 140px;
+  width: 325px;
+  margin-left: 20px;
+  position: absolute;
+}
+
+form {
+  position: absolute;
+  margin-top: 280px;
+  padding: 20px;
+  width: 375px;
+}
+
+#username,
+#email {
+  width: 327px;
+  height: 48px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  margin-bottom: 20px;
+  color: #172b4d;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  padding-left: 5px;
+}
+
+#textarea {
+  width: 327px;
+  height: 170px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  color: #b3bac5;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  padding: 5px;
+  margin-bottom: 20px;
+}
+
+.contact-btn {
+  border-radius: 8px;
+  background: #fff;
+  padding: 12px 16px;
+  color: #6070ff;
+  text-align: center;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.51px;
+  border: 1px solid #fff;
+}
+
```

Comment:
- [ ] Please make sure that your contact form button has interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template.

Reply:
You did well in implementing the button hover property, kindly use the design guideines for this so your design matches the design given 😄 

Figma design :
![image](https://github.com/garikaipaidamoyo/Portfolio/assets/20537712/3c0c6449-2133-4bae-aabe-27f95f1e6640)

Your design :

https://github.com/garikaipaidamoyo/Portfolio/assets/20537712/aa4aed0d-8ad9-4ed5-b636-56775018b140